### DISCUSSION TITLE
Departed Terrance changed more similiar to Cloister Prime

### DISCUSSION THREAD

**Discussion by Omega4200** [Crowns-of-Power/CoP#9] [Ideas]
DT bosses are on a 36hour timer best guild normally hoggs the bosses "what choice did i have :)" I think if it were spawn base bosses like CLoister prime would give others better chances. Maybe if we did that we wouldnt have a reason to play some Aggro wars that we all hated from back in the day. I would look into how hard/easy it is to spawn the CP bosses..... maybe make it a tiny bit harder to spawn the dt bosses

**Comment by allanon-pilgrim** [comment]
I like this, it would open it up for more people/groups. May still need a minimum respawn timer (maybe 2-3 hours) on top of the % spawn chance. Drop rates would probably also have to be reduced.

Question : 
Would it need to be possible to bypass some bosses?  Atm you kinda have to do them in order (yes I know you can run past them, using Bind Soul). CP you only enter the rooms you want to kill the bosses in. This could be done just by changing the position of boss/PH.

**Comment by gothrowhell2** [comment]
im actually down with this. we were looking into so many diffrent methods of stopping this from happening but honestly this sounds like close to the best without changing the entireity of cop. 👍

**Reply by 7hestalex** [reply]
yeah this is a good suggestion, and id say being able to skip certain PH / bosses would proly be a good idea if your a 5 man group you should be able to make it past bosses like twins if you cant 5 man them

also if its gonna be like most bosses where you kill PH then there shouldnt be a need for a cooldown for 2-3 hours as sugested by allanon, RNG should take care of it and if you get a double spawn then your just lucky

**Reply by TurtleScribe** [reply]
I’m with the boss skipping mechanic but the not replacing of the timer. If the timer is replaced it makes it possible for people to grind all the gear in a matter of days. The timer slows that down. It’s good
